rubi/rubi/client.py

```diff
@@ -380,9 +386,12 @@ def _default_event_handler(self, pair_name: str, event_type: Type[BaseEvent], ev
         if raw_event.client_filter(wallet=self.wallet):
             pair = self._pairs.get(pair_name)
 
-            order_event = OrderEvent.from_event(pair=pair, event=raw_event, wallet=self.wallet)
+            if isinstance(raw_event, EmitFeeEvent):
+                event = FeeEvent.from_event(pair=pair, event=raw_event)
```

Comment:
so i think open question for me here is if we want to keep these `FeeEvents` separated out from the `OrderEvent`. since `emitFee` includes the `offer id` that it corresponds to i think we handle it within the `OrderEvent` - especially since this event is emitted whenever a `maker rebate` occurs. guess its worth noting that this is also called when the `protocol` fee is collected. don't think its critical design choice, but could have implications on `rebate` tracking